scripts/cleanup_edomain_templates.py

```diff
@@ -73,13 +75,16 @@ def change_edomain_state(api, state, edomain):
                 elif state == 'active' and export_domain.get_status().state != 'active':
                     dc.storagedomains.get(edomain).activate()
 
-                wait_for(is_edomain_template_deleted,
-                         [api, state, edomain], fail_condition=False, delay=5)
-                print('{} successfully set to {} state'.format(edomain, state))
+                wait_for(is_edomain_in_state,
+                        [api, state, edomain], fail_condition=False, delay=5)
```

Comment:
Curious as to why we need to pass in api here. Can wrapanapi not be extended to include this function?